java/code/src/com/redhat/rhn/frontend/action/errata/ChannelAction.java

```diff
@@ -246,18 +246,20 @@ protected void processMethodKeys(Map<String, String> map) {
      * {@inheritDoc}
      * Add eid to our parameter map
      */
-    protected void processParamMap(ActionForm formIn,
+    public void processParamMap(ActionForm formIn,
                                    HttpServletRequest request,
                                    Map<String, Object> params) {
         //keep eid in params
-        Long eid = new RequestContext(request).getRequiredParam("eid");
+        Long eid = new RequestContext(request).getParamAsLong("eid");
```

Comment:
makes sense to use com.redhat.rhn.frontend.struts.RequestContext.ERRATA_ID here as well

Reply:
Addressed